leshan-bsserver-demo/src/main/resources/webapp/tag/bootstrap-modal.tag

```diff
@@ -147,7 +146,7 @@
                         smsBindingKeySecret : [  ],
                         smsSecurityMode : "NO_SEC",
                         uri : bsserver.uri,
-                        oscoreSecurityMode : 111 // link to oscore object
+                        oscoreSecurityMode : 0 // link to bs oscore object
```

Comment:
This value should only be set if `oscore` is used  ?

Reply:
Updated in commit b56a24f02171a0d81693d6e944a4b69bd22026a0